src/focus.ts

```diff
@@ -0,0 +1,40 @@
+export function focusElement(elementOrSelector: string | HTMLElement) {
+	let el: HTMLElement | null;
+	if (typeof elementOrSelector === 'string') {
+		el = document.querySelector<HTMLElement>(elementOrSelector);
+	} else {
+		el = elementOrSelector;
+	}
+
+	if (!(el instanceof HTMLElement)) return;
+
+	// Set and restore tabindex to allow focusing non-focusable elements
```

Comment:
We aren't restoring the tabindex anymore. I'd suggest `// Apply a tabindex attribute to the element to allow it to be focused`

Reply:
Right, that was outdated. Changed to something in between.